src/pdn/README.md

```diff
@@ -20,26 +20,26 @@ be defined over the stdcell area, and over areas occupied by macros.
 Build a power grid in accordance with the information specified.
 
 ```tcl
-pdngen 
-    [-skip_trim]
+pdngen
     [-dont_add_pins]
+    [-failed_via_report file]
+    [-report_only]
     [-reset]
     [-ripup]
-    [-report_only]
-    [-failed_via_report file]
+    [-skip_trim]
     [-verbose]
```

Comment:
-verbose is missing in the list of options below

Reply:
Yes I saw it. I saw other issues too. I'll fix them in another PR. This PR only sorts args.